### ISSUE TITLE
Blue Griffon corrupts <xmp> tag contents by encoding the > & < characters, causing incorrect rendering in browsers.

### ISSUE THREAD

**Issue by snovotill** [therealglazou/bluegriffon#109]
The <xmp> tag is still fully supported by FireFox and Chrome as of 2024 but it is fully unusable within BlueGriffon because BlueGriffon will replace the characters > & < with &gt; &amp; &lt; whenever switching between Source and Wysiwyg views. This causes the characters to render incorrectly within FireFox and Chrome. This is only relevant to the <xmp> tag and no other HTML tags exhibit this behavior.

In other words, `<xmp> Hello <world> & be good!<xmp>` must NOT be changed to `<xmp> Hello &lt;world&gt; &amp; be good!<xmp>` when inside the <xmp tag, because it will then render INCORRECTLY in FireFox and Chrome.
